service/serviceconfig/service.go

```diff
@@ -114,32 +120,16 @@ type ServiceTelemetryMetrics struct {
 	Address string `mapstructure:"address"`
 }
 
-func (s ServiceTelemetryMetrics) validate() error {
+func (s ServiceTelemetryMetrics) Validate() error {
```

Comment:
Do these need to be exported?  I don't see them used anywhere else.

Reply:
Done.